Dockerfile.cross

```diff
@@ -10,8 +10,11 @@ RUN __DEPENDENCIES__
 
 ENV PKG_CACHE_PATH=/fetched
 
-RUN mkdir -p /fetched && npm install pkg-fetch
+RUN git clone -b patch-1 --single-branch https://github.com/n1ru4l/pkg-fetch.git /build
+WORKDIR /build
+RUN yarn && yarn run babel
+RUN mkdir -p /fetched
```

Comment:
`npm install --save n1ru4l/pkg-fetch#patch-1` much cleaner

Reply:
But it will take a lot longer as the whole repository is cloned.

Reply:
Didn't know that, ok keep this so :) Why are you using yarn and not npm?

Reply:
Because `pkg-fetch` has a `yarn.lock` file. https://github.com/vercel/pkg-fetch/blob/master/yarn.lock

I did not want to add some random failure condition :D

Reply:
@n1ru4l also missed that, let me go take a coffie :coffee: :laughing:

Reply:
@n1ru4l Did you tested it?

Reply:
Did abort the build. But I can run it again and check the result. Any estimation on how long it will take? 🙄

Reply:
I just restarted and hope it will not take 12 hours like the GitHub actions 🤣. What hardware are you using for the github action runner?

Reply:
I use github actions with a custom runner (an Intel NUC I have at work). Usually a build takes around 24 hours :laughing:

Reply:
To trigger new builds I just add a comment to this issue: https://github.com/robertsLando/pkg-binaries/issues/5

Reply:
Ok, lets see how long it will take on my MacBook :D

Reply:
It is already taking 2 hours 😅 Does building node always take that long? Even on "powerful" machines? :o

Reply:
> Ok, lets see how long it will take on my MacBook :D I guess normal GitHub action is not an option as it will time out eventually?

This is why I have setup a self-hosted runner, github max timeout is about 6 hours (https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners)

Reply:
> It is already taking 2 hours sweat_smile Does building node always take that long? Even on "powerful" machines? :o

Sincerly I dunno, It has always take so long to build even on my powerful machines, some hours or even a day when using emulated environments with docker buildx

Reply:
Okay it seems like QEMU in Gerberas so super slow. There are other options available though: https://medium.com/nttlabs/buildx-multiarch-2c6c2df00ca2

Reply:
Ok, I aborted local build and set up a GitHub action runner on one of my existing servers 😅.

check https://github.com/dungeon-revealer/pkg-binaries/runs/1004548703?check_suite_focus=true

Hopefully, it will not take TOO long.

Reply:
@n1ru4l let me know :)

Reply:
So the first build failed during the night due to `rename` not being shipped with ubuntu by default. Build took **only** ~11 hours.

Reply:
"only"

Reply:
It is also possible to speed up the build by using more cores/threads: https://github.com/dungeon-revealer/pkg-binaries/commit/44328747c38a91116308af5c23859dc94b3cad36